spock-core/src/main/java/org/spockframework/compiler/AstNodeCache.java

```diff
@@ -81,7 +118,50 @@
   // mocking API
   public final ClassNode InteractionBuilder = ClassHelper.makeWithoutCaching(org.spockframework.mock.runtime.InteractionBuilder.class);
 
+  public final MethodNode InteractionBuilder_SetRangeCount =
+    InteractionBuilder.getDeclaredMethods(org.spockframework.mock.runtime.InteractionBuilder.SET_RANGE_COUNT).get(0);
+  public final MethodNode InteractionBuilder_SetFixedCount =
+    InteractionBuilder.getDeclaredMethods(org.spockframework.mock.runtime.InteractionBuilder.SET_FIXED_COUNT).get(0);
+  public final MethodNode InteractionBuilder_AddEqualTarget =
+    InteractionBuilder.getDeclaredMethods(org.spockframework.mock.runtime.InteractionBuilder.ADD_EQUAL_TARGET).get(0);
+  public final MethodNode InteractionBuilder_AddWildcardTarget =
+    InteractionBuilder.getDeclaredMethods(org.spockframework.mock.runtime.InteractionBuilder.ADD_WILDCARD_TARGET).get(0);
+  public final MethodNode InteractionBuilder_AddEqualMethodName =
+    InteractionBuilder.getDeclaredMethods(org.spockframework.mock.runtime.InteractionBuilder.ADD_EQUAL_METHOD_NAME).get(0);
+  public final MethodNode InteractionBuilder_AddRegexMethodName =
+    InteractionBuilder.getDeclaredMethods(org.spockframework.mock.runtime.InteractionBuilder.ADD_REGEX_METHOD_NAME).get(0);
+  public final MethodNode InteractionBuilder_AddEqualPropertyName =
+    InteractionBuilder.getDeclaredMethods(org.spockframework.mock.runtime.InteractionBuilder.ADD_EQUAL_PROPERTY_NAME).get(0);
+  public final MethodNode InteractionBuilder_AddRegexPropertyName =
+    InteractionBuilder.getDeclaredMethods(org.spockframework.mock.runtime.InteractionBuilder.ADD_REGEX_PROPERTY_NAME).get(0);
+  public final MethodNode InteractionBuilder_SetArgListKind_boolean_boolean =
```

Comment:
This seems to be the only instance where the parameters are listed.

Reply:
Because it is the only method that has overloads.